versionedRouter.js

```diff
@@ -666,6 +668,9 @@ if (transformerProxy) {
         async ctx => {
           const startTime = new Date();
           ctx.set("apiVersion", API_VERSION);
+          const reqOpts = {
+            timeout: ctx.get('RdProxy-Timeout')
+          };
```

Comment:
`handleProxyRequest` already has access to `ctx`. I don't see a need to do this here.

Reply:
If you observe keenly, we don't use `ctx` inside the methods that we're writing tests on!
Followed the same pattern. Let me know if there is a disagreement

Reply:
I don't think that's the case. Even if it was, doesn't justify it.
Unless there is any other compelling reason, I'd fix this.

Reply:
I disagree with the intention here. It doesn't make sense to include the CTX(which is `koa.Context`) in tests as it is more framework and not the ruddestack logic, upon which we'll need to write the tests
Probably it is wrong to send `ctx` as an argument to `handleProxyRequest`, we should may be pass required details only to `handleProxyRequest`

Reply:
@sanpj2292 I agree with you sending whole context to handleProxyRequest  makes the function more complex to test so get rid of it.

Reply:
@koladilip Done. Can you please check and let me know if anything else is required ?